File: model1/src/main/java/com/yxy/model1/Module1Application.java
package com.yxy.model1;

import android.util.Log;

import com.yxy.common.AppBase;
import com.yxy.structure.core.ModuleSpec;

//priority值越大越后执行
@ModuleSpec(priority = 2)
public class Module1Application extends AppBase {
    @Override
    public void onCreate() {
        super.onCreate();
        Log.i("tag5","Module1Application  oncreate");
    }
}

File: model1/src/main/java/com/yxy/model1/Module1Application.kt
package com.yxy.model1

import android.util.Log
import com.yxy.common.AppBase
import com.yxy.structure.core.ModuleSpec


@ModuleSpec(priority = 2)
open class Module1Application : AppBase() {

  override fun onCreate() {
    super.onCreate()
    // do module1 initialization
    Log.i("module1", "module1 init is called")
  }
}
